--- src/Mp3TransformThread.cpp.orig	2024-06-25 10:25:32.868223466 +0200
+++ src/Mp3TransformThread.cpp	2024-06-25 10:25:56.937243820 +0200
@@ -676,8 +676,8 @@
     {
         qDebug("Caught std::exception in Mp3TransformThread::transform()");
         traceToFile("Caught std::exception in Mp3TransformThread::transform()", 0);
-        qDebug("%s", ex.what());
-        traceToFile(ex.what(), 0);
+        qDebug("%s", "ex.what()");
+        traceToFile("ex.what()", 0);
         throw; // !!! needed to restore "erased" files when errors occur, because when an exception is thrown the destructors only get called if that exception is caught; so catching and rethrowing is not a "no-op"
     }
     catch (...)
